Src/AutoFixture.NUnit3.UnitTest/PilotTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class PilotTest
+    {
+        [Test]
+        public void AssertTrue()
+        {
+            Assert.IsTrue(true);
```

Comment:
It's not that it's a big deal, but I prefer only committing code to the repository if it has a long-term purpose.

I do realise that it's temporary, but the problem with committing temporary code is that it may become permanent.

While _you_ may know that you'd like to remove this file again later, other people may not be aware of this. It's probably going to be fine, but what _could_ happen is that after you've committed this file, something happens to you that sidetracks you:

- You may become very busy at work
- You win the lottery, and decide to travel the world instead of contributing to open source
- You meet a nice person (if you're not already in a relationship)
- You're in an accident
- Etc.

If we _can't avoid_ temporary things, we'll take them, but if they _can_ be avoided, I'd prefer to do so, as it minimises the items that everyone needs to keep track of.

If a temporary item _can't_ be avoided, I'd prefer a big fat comment to that effect in the source file, if at all possible.